### PR TITLE
#1750 Expose API test command in manager's yarn command

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1750)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/scripts/runE2eTests.js [hunk 1]
```diff
@@ -129,7 +129,7 @@ const run = async (prod, apiRoot, serverPort = 5050, clientPort = 3000) => {
   cypress = execa(
     'yarn',
     [
-      'e2e:run',
+      'test:e2e:run',
```
Comment (wu87988622) [inline ohara-manager/scripts/runE2eTests.js]: Are you miss runApiTests.js some change ?

Comment (konekoya) [reply ohara-manager/scripts/runE2eTests.js]: Yep. But I'm wondering why it's still working...

**Comment by konekoya** [comment]
retry manager-it

**Comment by konekoya** [comment]
retry manager-api

**Comment by konekoya** [comment]
retry manager-api

**Comment by wu87988622** [comment]
> retry manager-api

Look the api test is working !!!
Thanks

**Comment by chia7712** [comment]
嗯？這個變更的目的是什麼？

**Comment by konekoya** [comment]
> 嗯？這個變更的目的是什麼？

So that we can run the command from the `ohara-manager/` not `ohara-manager/client` :)

**Comment by konekoya** [comment]
retry manager-it

**Comment by konekoya** [comment]
retry manager-it

**Comment by konekoya** [comment]
retry manager-api

**Comment by konekoya** [comment]
retry manager-it

**Comment by konekoya** [comment]
retry manager-it

**Comment by konekoya** [comment]
retry manager-it

**Comment by konekoya** [comment]
retry manager-it

**Comment by konekoya** [comment]
retry manager-it

**Comment by konekoya** [comment]
retry manager-it

**Pull request merged**